### PR TITLE
Add situ data api source and added meta_data to in situ database documents

### PR DESCRIPTION
**Pull request by Ben-EJ**

### CODE REVIEW

File: air-quality-backend/src/etl/in_situ/openaq_adapter.py [hunk 1]
```diff
@@ -50,9 +50,11 @@ def transform_in_situ_data(in_situ_data):
     return _calculate_overall_aqi_value(formatted_dataset)
 
 
+#    "entity", "sensorType" group this stuff into its own object (called meta data)
```
Comment (amehta-scottlogic) [inline air-quality-backend/src/etl/in_situ/openaq_adapter.py:53]: Please remove this comment

File: air-quality-backend/src/etl/in_situ/openaq_adapter.py [hunk 2]
```diff
@@ -63,6 +65,10 @@ def _create_document(measurement, city_name):
         "measurement_date": datetime.strptime(
             measurement["date"]["utc"], "%Y-%m-%dT%H:%M:%S%z"
         ),
+        "meta_data": {
```
Comment (amehta-scottlogic) [inline air-quality-backend/src/etl/in_situ/openaq_adapter.py:68]: lets have metadata be one word

File: air-quality-backend/src/etl/in_situ/openaq_adapter.py [hunk 3]
```diff
@@ -63,6 +65,10 @@ def _create_document(measurement, city_name):
...
+            "entity": measurement["entity"],
+            "sensorType": measurement["sensorType"],
```
Comment (amehta-scottlogic) [inline air-quality-backend/src/etl/in_situ/openaq_adapter.py:70]: Can you put a screenshot showing the data in the database after it runs?

**Review by amehta-scottlogic — CHANGES_REQUESTED** [review]

File: air-quality-backend/src/etl/in_situ/openaq_adapter.py (hunk 3 above)
Comment (Ben-EJ) [reply air-quality-backend/src/etl/in_situ/openaq_adapter.py:70]: image

**Review by amehta-scottlogic — DISMISSED** [review]

**Review by amehta-scottlogic — APPROVED** [review]

**Pull request merged**